src/main/java/com/royalrangers/controller/UserController.java

```diff
@@ -83,8 +83,21 @@ public ResponseResult rejectUser(@RequestBody IdsDto param) {
         return ResponseBuilder.success("Users successfully rejected.");
     }
 
-    @PutMapping
-    @ApiOperation(value = "Update current user")
+    @PutMapping("/update/temp")
+    @ApiOperation(value = "Update temp user")
```

Comment:
Change to "Update user data (for users). It will be in temp_user table until admin will approve it.